src/LaravelBook/Ardent/Ardent.php

```diff
@@ -885,7 +885,7 @@ public static function find($id, $columns = array('*')) {
 	 * @return \Illuminate\Database\Eloquent\Builder
 	 */
 	public function newQuery($excludeDeleted = true) {
-		$builder = new Builder($this->newBaseQueryBuilder());
+		$builder = parent::newQuery();
```

Comment:
would stop the library of using it's own `Builder`. Maybe `newEloquentBuilder` should be overridden?